src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -116,10 +115,9 @@
 (defn discover []
   (let  [view-type (reagent/atom :card-view)]
     (fn []
-      (let [communities (rf/sub [:communities/communities])
-            communities-count (count communities)
-        ; TODO move sorting to subscription
```

Comment:
actually, do we really want sorting in the subs? This seems like unnecessary computation we probably don't want for composing subs.. Right @ilmotta , @cammellos , @flexsurfer ? 🤔

Reply:
actually its my comment, views should be dumb, all computation should be in subscriptions, so there should be `:communities/sorted-discover-communities` subscription

Reply:
I prefer to think that sorting is not the view responsibility. The view just cares about getting the data and transforming it to Hiccup.

If we move the sorting logic out of the subscription, we actually lose the caching facilities because now sort will be computed on every render (and they happen a lot).